core/src/main/resources/schema/app/20180504144105_BUG-102208_remove_nonnull_from_nodecount.sql

```diff
@@ -0,0 +1,16 @@
+-- // BUG-102208 remove nonnull from nodecount
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE instancegroup ALTER nodecount DROP NOT NULL;
+
+-- //@UNDO
+-- SQL to undo the change goes here.
+
```

Comment:
I think first we need an ALTER TABLE for adding the previously deleted column back to the table, don't we?

Reply:
only not null constraint is dropped, not the column:
... ALTER nodecount DROP NOT NULL